impl/java/test/src/main/kotlin/br/com/guiabolso/events/test/GeneralEventsMatchers.kt

```diff
@@ -55,40 +55,40 @@ fun haveFlowId(flowId: String) = object : Matcher<Event> {
 }
 
 inline fun <reified T> Event.shouldContainPayload(key: String, value: T) =
-    payload.toString().shouldContainJsonKeyValue(key, value)
+    payload.toJson().shouldContainJsonKeyValue(key, value)
```

Comment:
Revert to use toString() since JsonNode will produce a valid json on toString() function.